Assets/Scripts/Entities/Entity.cs

```diff
@@ -22,7 +22,10 @@ public Vector2 position
     public abstract void Update();
 
     // This is where input is received
-    public abstract void HandleInstruction(Instruction instruction);
+    public virtual void HandleInstruction(Instruction instruction)
+    {
+        this.transform.position = GameManager.GetTransformForInstruction(instruction, transform.position);
```

Comment:
Sweet. This will probably eventually come from some kind of InputManager or something to that effect, which just handles inputs from the user.